source/tutorials/pnp-testsuites/pnp-testsuites.rst

```diff
@@ -0,0 +1,545 @@
+
+Test Suite
+==========
+
+This section describes how to run the workloads included in the |C| Power and Performance (PnP)
+test suite and the Key Performance Indicators (KPIs) measured during the tests.
+
+H.264 3840x2160 60fps Playback
+------------------------------
+
+This KPI measures frame rate for video playback of a locally stored video clip using HSC.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Set up HSC environment.
+
+**Measurement Steps**
+
+    #. Download the H.264 3840x2160 60fps video source and push it into the device.
+
+    #. Play the video using the Gallery application or other video playback application.
+
+    #. Calculate FPS using HSC.
+
+Peak 1080p H.264 Decoding FPS
+-----------------------------
+
+This KPI measures the capability of parallel multiway decoding using an MSDK sample.
+For example, this could be built with the command: 
+
+.. code-block:: bash
+
+    $ make -j8 BOARD_HAVE_MEDIASDK_OPEN_SOURCE=true sample_decode
+
+The output is located in :file:`out/target/product/cel_kbl/vendor/bin` folder.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Compile or download the decoder sample file.
+
+    #. Download or create a H.264 testing video clip (FFmpeg can be used).
+
+**Measurement Steps**
+
+    #. Push the H.264 video clip and the compiled decoder sample binary file to the test device.
+
+    #. Set the decoder binary file executable, and execute the decoder sample binary file to get FPS.
+       Since the decoder program may not fully utilize 100% of the decoding engine's capability
+       on many platform, it's recommended to create a script to run multiple (e.g. 4) instances
+       of the decoding program in parallel, and aggregate the reported FPS numbers
+       as the final result.
+
+Peak 1080p H.264 Encoding FPS
+-----------------------------
+
+This KPI measures the capability of parallel multiway encoding using an MSDK sample. 
+This could be built using the command:
+
+.. code-block:: bash
+
+    $ make -j8 BOARD_HAVE_MEDIASDK_OPEN_SOURCE=true sample_encode
+
+The output is located in :file:`out/target/product/cel_kbl/vendor/bin` folder.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Compile or download the encoder sample file.
+
+    #. Download or create a 1080p testing YUV video clip (FFmpeg can be used).
+
+**Measurement Steps**
+
+    #. Push the YUV video clip and the compiled encoder sample binary file to the test device.
+
+    #. Set the encoder binary file executable, and execute the encoder sample binary file to get FPS.
+       Since the encoder program may not fully utilize 100% of the eecoding engine's capability
+       on many platform, it's recommended to create a script to run multiple (e.g. 4) instances
```

Comment:
Change to:
Since the encoder program may not fully utilize 100% of the encoding engine's capability on many platforms, it's recommended to create a script to run multiple instances (e.g. 4)